package.json

```diff
@@ -29,6 +33,7 @@
 	"dependencies": {},
 	"devDependencies": {
 		"ava": "^6.1.3",
+		"tsd": "^0.31.1",
```

Comment:
Most packages don't actually need type-tests. They just need to validate the types, so I prefer to just run typescript on the types.